buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+  `kotlin-dsl`
+}
+
+dependencies {
+  implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.0")
+  implementation("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:3.5.0.2730")
+  implementation("org.jetbrains.dokka:dokka-gradle-plugin:1.7.20")
+}
```

Comment:
I like using `buildSrc/build.gradle.kts` as a centralised place to define plugin versions, so that there's no need to define the plugin versions anywhere else. 

I didn't add all of the plugins here, but I could if you wanted?

Reply:
Might as well. It'll be cleaner.

Reply:
Done! I had to bump the Kover version, I think the coordinates have changed and it was easier to bump the version than try and find the old coordinates.

Reply:
@aSemy  Great, Thanks.